src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -12,27 +12,18 @@ using namespace winrt::Windows::UI::Xaml::Hosting;
 using namespace winrt::Windows::Foundation::Numerics;
 using namespace ::Microsoft::Console::Types;
 
-// The tabs are 34.8px tall. This is their default height - we're not
-// controlling the styling of the tabs at all currently. If we change the size
-// of those, we'll need to change the size here, too. We can't get this size
-// from the tab control until the control is added to a XAML element, and we
-// can't create any XAML elements until we have a window, and we need to know
-// this size before we can create a window, so unfortunately we're stuck
-// hardcoding this.
-const int NON_CLIENT_CONTENT_HEIGHT = static_cast<int>(std::round(34.8));
+double NON_CLIENT_DRAGBAR_WIDTH = 0.0;
 
 AppHost::AppHost() noexcept :
     _app{},
     _window{ nullptr }
 {
     _useNonClientArea = _app.GetShowTabsInTitlebar();
+    NON_CLIENT_DRAGBAR_WIDTH = _app.GetNonClientAreaDragBarWidth();

```

Comment:
This is weird. If it's not a constant, name it normally: without all caps and underscores.